html/global_attributes.json

```diff
@@ -176,18 +176,18 @@
             },
             "samsunginternet_android": [
               {
+                "version_added": "9.0",
                 "notes": [
                   "Support added for <code>autocomplete</code> on the <code>&lt;textarea&gt;</code> and <code>&lt;select&gt;</code> elements.",
-                  "Samsung Internet does not accept <code>off</code> as a value.  See <a href='https://crbug.com/587466'>bug 587466</a>."
-                ],
-                "version_added": "9.0"
+                  "Chrome does not accept <code>off</code> as a value.  See <a href='https://crbug.com/587466'>bug 587466</a>."
```

Comment:
This change seems to just be a potential issue with the notes conversion in the mirroring script.  Can we revert this change?